### DISCUSSION TITLE
How do I work with subdomains if the subdomain repo is in another github repo?

### DISCUSSION THREAD

**Discussion by Rupsnigdha** [vercel/community#5635] [Feature Question] [answered]
### Question

I built a site, and now I need to build the admin panel. I realise that I am able to create a subdomain easily using Vercel. However, the tech stack for the admin panel is in a different repo, and I don't see an option to point to a different repo for the subdomain. How do I proceed from here? Any help is appreciated! Thanks!!

**Comment by TooTallNate** [comment, marked as answer]
You Admin panel app should be its own Project on Vercel, which allows you to configure the subdomain and the repo from the Project Settings.
